stage0/entrypoint.go

```diff
@@ -26,9 +26,9 @@ import (
 )
 
 const (
-	enterEntrypoint = "coreos.com/rkt/stage1/enter"
-	runEntrypoint   = "coreos.com/rkt/stage1/run"
-	gcEntrypoint    = "coreos.com/rkt/stage1/gc"
+	enterEntrypoint = "stage1-enter"
+	runEntrypoint   = "stage1-run"
+	gcEntrypoint    = "stage1-gc"
```

Comment:
Please also update the docs as you said yourself in the following comment ;)
https://github.com/coreos/rkt/pull/1037#commitcomment-11658569